port/raspberrypi/rp2xxx/src/hal/cyw43_pio_spi.zig

```diff
@@ -0,0 +1,227 @@
+//! CYW43 3-wire SPI driver build on PIO
+//! Code based on embassy cyw43-pio driver https://github.com/embassy-rs/embassy/tree/main/cyw43-pio
```

Comment:
Maybe we should put a tag or commit instead of main here since main is a moving target